setup.cfg

```diff
@@ -1,18 +1,25 @@
 [metadata]
-name = ctrl_mpexec
+name = lsst-ctrl-mpexec
 description = Pipeline execution infrastructure for the Rubin Science Pipelines
 author = Rubin Observatory Data Management
 url = https://github.com/lsst/resources
```

Comment:
Wrong URL

Reply:
So this should use solely python 3.10?

Reply:
I think you are replying to the other comment...